Lib/fontTools/varLib/mutator.py

```diff
@@ -145,7 +145,7 @@ def interpolate_cff2_metrics(varfont, topDict, glyphOrder, loc):
 			hmtx[gname] = tuple(entry)
 
 
-def instantiateVariableFont(varfont, location, inplace=False):
+def instantiateVariableFont(varfont, location, inplace=False, set_macos_overlap_rendering_bit=False):
```

Comment:
I would call the option `overlapping_contours`

Reply:
Will do!

Reply:
https://github.com/fonttools/fonttools/pull/1518/commits/fee4848d4ae1ee0d69d368cc4d731423172111b3